src/providers/linkedin.ts

```diff
@@ -59,14 +65,30 @@ export class LinkedIn implements OAuth2Provider {
 	}
 }
 
-interface TokenResponseBody {
+interface AuthorizationCodeResponseBody {
```

Comment:
Can you separate this into `AuthorizationCodeResponseBody` and `RefreshTokenResponseBody` to be consistent with other providers?

Reply:
I have already separated it into `AuthorizationCodeResponseBody` and `RefreshTokenResponseBody`.

Reply:
Oh wait, my bad!

Reply:
No worries! Thanks for the wonderful library!